### PR TITLE
Fix product variation tooltip

### LINKED ISSUE #33552: Product creation experience: variation - default form value - tooltip not working
**Description:** the "tooltip displayed alongside the default form value option is not working
Captura de ecrã 2022-06-22, às 11 43 57

- Tooltip does not appear before Draft is saved

**Acceptance criteria:**
- [ ] Fix the tooltip appearing
- [ ] Update copy: Choose a default form value if you want a certain variation already selected when a user visits the product page

### PR DESCRIPTION
**Pull request by louwie17**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR updates the copy of the default form values tooltip shown in the variable product meta box and also fixes the tooltip initialization when no variations exist yet.

Closes #33552 .

### How to test the changes in this Pull Request:

1. Load this PR and do a build, make sure the `woocommerce/client/legacy` is freshly build, I had to run `pnpm -- turbo run build --filter=woocommerce/client/legacy --force` to make sure the changes propagated instead of it using the cached version.
2. Go to **Products > Attributes** and add some attributes with terms
3. Go to **Products > Add New** and change the product to **Variable product**
4. Go to the **Attributes** tab and add the attribute you created in step 2, make sure to select **Used for variations** and click **Save attributes** after (you have to add values here)
5. Go to the **Variations** tab and click **Go** for the **Add variation** selection.
6. It should now show a **Default Form Values** with a tooltip after (as shown in the screenshot), the tooltip should work as expected
7. The tooltip should say: `Choose a default form value if you want a certain variation already selected when a user visits the product page`

Screen Shot 2022-07-05 at 4 31 11 PM

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [x] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js [hunk 1]
```diff
@@ -118,7 +118,13 @@ jQuery( function( $ ) {
 			// Init TipTip
 			$( '#tiptip_holder' ).removeAttr( 'style' );
 			$( '#tiptip_arrow' ).removeAttr( 'style' );
-			$( '.woocommerce_variations .tips, .woocommerce_variations .help_tip, .woocommerce_variations .woocommerce-help-tip', wrapper )
+			$(
+				'.woocommerce_variations .tips, ' +
+				'.woocommerce_variations .help_tip, ' +
+				'.woocommerce_variations .woocommerce-help-tip, ' +
+				'.toolbar-variations-defaults .woocommerce-help-tip',
```
Comment (louwie17) [inline plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js:125]: Added this `'.toolbar-variations-defaults .woocommerce-help-tip'` to make sure that the default form value tooltip was also being initialized.

**Comment by joelclimbsthings** [comment]
Thanks @louwie17 , code changes look good and the tooltip is working. One issue that may or may not be worth fixing depending on how involved it would be; if you approach the tooltip from the bottom you can pretty easily get a flashing behavior:

Peek 2022-07-05 17-24

Saw it on both Firefox and Chrome, and haven't seen it on other tooltips.

**Review by joelclimbsthings — CHANGES_REQUESTED** [review]
Thanks @louwie17 , code changes look good and the tooltip is working. One issue that may or may not be worth fixing depending on how involved it would be; if you approach the tooltip from the bottom you can pretty easily get a flashing behavior:

Peek 2022-07-05 17-24

Saw it on both Firefox and Chrome, and haven't seen it on other tooltips.

**Comment by louwie17** [comment]
> Thanks @louwie17 , code changes look good and the tooltip is working. One issue that may or may not be worth fixing depending on how involved it would be; if you approach the tooltip from the bottom you can pretty easily get a flashing behavior:

Thanks for the review @joelclimbsthings, I was able to reproduce the flickering, nice catch! Adding `pointer-events: none` to the tooltip css seemed to do the trick and still keeps the tooltip working as expected.
This is ready for a re-review if you have some time @joelclimbsthings :)

**Review by joelclimbsthings — APPROVED** [review]
Thanks @louwie17 , had to remember to clear the turbo cache again, but it worked great! Nice solution. 🚢

**Pull request merged**